### ISSUE TITLE
Searching for a subreddit crashes the app

### ISSUE THREAD

**Issue by helloyanis** [Docile-Alligator/Infinity-For-Reddit#1665]
### Checklist

- [X] I have used the search function for [open](https://github.com/Docile-Alligator/Infinity-For-Reddit/issues) **and** [closed](https://github.com/Docile-Alligator/Infinity-For-Reddit/issues?q=is%3Aissue+is%3Aclosed) issues to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.
- [X] If the bug only occurs with a certain link, post, image..., I will include the URL.

### App version

7.2.4-beta1

### Where did you get the app from

Google Play

### Android version

14

### Device model

Samsung Galaxy A52S 5G

### First occurred

Yesterday but I don't check often

### Steps to reproduce

1. Open Infinity
2. Search for anything using the search bar
3. Go to the subreddit tab
4. App crashes

### Expected behaviour

App displays subreddit list

### Current behaviour

App crashes
https://github.com/Docile-Alligator/Infinity-For-Reddit/assets/40769776/327a7ee6-4e0e-4014-b8e6-2ff1ab6321ef

### Logs

```
`` Stacktrace ``

java.lang.RuntimeException: Can't create handler inside thread Thread[arch_disk_io_3,5,main] that has not called Looper.prepare()
	at android.os.Handler.<init>(Handler.java:227)
	at android.os.Handler.<init>(Handler.java:129)
	at Q6.s.<init>(SourceFile:20)
	at Q6.t.b(SourceFile:18)
	at M1.n$c$a.a(SourceFile:7)
	at M1.n$c$a.e(SourceFile:1)
	at M1.A0.d(SourceFile:3)
	at M1.A0.e(SourceFile:1)
	at M1.G$b.y(SourceFile:71)
	at L5.a.i(SourceFile:12)
	at b6.V.run(SourceFile:99)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:644)
	at java.lang.Thread.run(Thread.java:1012)
	Suppressed: h6.i: [K0{Cancelling}@c1fa152, r.b@40adc23]


`` END of stacktrace ``

`` Thread info ``

Name: arch_disk_io_3
ID: 138
State: RUNNABLE
Priority: 5
Thread group name: main
Thread group parent: system
Thread group active count: 62
Thread time: 0 hr 0 min, 0 sec

`` END of thread info ``

`` Device info ``

Report ID: 01a7bc43-f4e5-4e55-ae60-2c6098f1f717
Device ID: 9e1f4cb679fe97ad
Application version: 170
Default launcher: com.sec.android.app.launcher
Timezone name: heure normale d’Europe centrale
Timezone ID: Europe/Paris
Version release: 14
Version incremental : A528BXXS6FXA1
Version SDK: 34
Board: lahaina
Bootloader: A528BXXS6FXA1
Brand: samsung
CPU ABIS 32: armeabi-v7a, armeabi
CPU ABIS 64: arm64-v8a
Supported ABIS: arm64-v8a, armeabi-v7a, armeabi
Device: a52sxq
Display: UP1A.231005.007.A528BXXS6FXA1
Fingerprint: samsung/a52sxqxx/a52sxq:11/RP1A.200720.012/A528BXXS6FXA1:user/release-keys
Hardware: qcom
Host: SWDK6216
ID: UP1A.231005.007
Manufacturer: samsung
Product: a52sxqeea
Build time: 1704189170000
Build time formatted: 02-01-2024 10:52:50
Type: user
Radio: A528BXXS6FXA1,A528BXXS6FXA1
Tags: release-keys
User: dpi
User IDs: N/A
Is sustained performance mode supported: No
Is in power save mode: No
Is in interactive state: Yes
Is ignoring battery optimizations: No
Thermal status: STATUS_NONE
Location power save mode: MODE_NO_CHANGE
Is device idle: No
Battery percentage: 74
Battery remaining time: 01-01-1970 01:00:00
Is battery charging: No
Is device rooted: No
CPU Model: N/A
Number of CPU cores: 8
Up time with sleep: 533 hr 34 min, 47 sec
Up time without sleep: 192 hr 2 min, 10 sec

`` END of Device info ``

`` Exit reasons ``

N/A

`` END of exit reasons ``

`` Application info ``

App name: Infinity
Version code: 170
Version name: 7.2.4-beta1
Package name: ml.docilealligator.infinityforreddit
Short package name: infinityforreddit
Flavor: N/A
Signatures: XFN80TR5rgSMbsH+WZ5oyxarmIU=
Is debuggable: No
First installed: 04-06-2024 12:49:38
Last updated: 04-06-2024 12:49:38
Requested permissions: android.permission.ACCESS_NETWORK_STATE, android.permission.INTERNET, android.permission.SET_WALLPAPER, android.permission.SET_WALLPAPER_HINTS, android.permission.FOREGROUND_SERVICE, android.permission.WAKE_LOCK, android.permission.POST_NOTIFICATIONS, android.permission.USE_BIOMETRIC, android.permission.USE_FINGERPRINT, android.permission.RECEIVE_BOOT_COMPLETED, ml.docilealligator.infinityforreddit.DYNAMIC_RECEIVER_NOT_EXPORTED_PERMISSION, com.android.vending.BILLING
Default prefs: N/A
Default prefs: N/A

`` END of Application info ``


`` Currently running foreground/background processes ``

N/A

`` END of running foreground/background processes info ``
```

**Comment by dezzo77** [comment]
Also having this issue. Infinity disappears when searching. No error message.

**Comment by Docile-Alligator** [comment]
This is fixed in v7.2.5.
